dev/config/keycloak/import/ltt-realm.json

```
@@ -112,7 +112,7 @@
        "alwaysDisplayInConsole": false,
        "clientAuthenticatorType": "client-secret",
        "redirectUris": [
-         ""
+         "*"
        ],
        "webOrigins": [],
```

Adding "*" to `webOrigins` is also necessary to get the token at `https://keycloak.inform.dev.cirg.uw.edu/realms/ltt/protocol/openid-connect/token`
```suggestion
       "webOrigins": [
         "*"
       ],
```

---

Thanks! Hadn't realized `webOrigins` (*Allowed CORS origins*`) was required for frontend OIDC clients